packages/front/src/routes/search/[name]/+page.svelte

```diff
@@ -24,7 +29,9 @@
         </div>
     </div>
 {:then _result}
-    <p class="fr-text-md">{`${$associations.length} résultats trouvés.`}</p>
+    <p class="fr-text-md">
+        {`${nbResultLabel}`}
```

Comment:
du coup 
```suggestion
        {nbResultLabel}
```
Le but de mon commentaire initial étant que c'était pas nécessaire de faire une string paramétrée dans le template svelte